.github/workflows/analytics.yml

```diff
@@ -57,6 +57,15 @@ jobs:
       run: |
         docker tag georchestra/analytics:${{ steps.version.outputs.VERSION }} georchestra/analytics:latest
         docker push georchestra/analytics:latest
+        
+    - name: "Update Docker Hub Description"
+      uses: peter-evans/dockerhub-description@v3
```

Comment:
You have to make this step ran only on the default branch.
Otherwise it will fail on external pull requests.

Same for the other same step `"Update Datafeeder Docker Hub Description"` in the other files.

Reply:
Indeed, but didn't put it by now to see if it pass.